app/views/devise/mailer/invitation_instructions.html.erb

```diff
@@ -1,8 +1,26 @@
+<%
+  tool_name = Rails.configuration.branding[:application][:name]
+  link = accept_invitation_url(@resource, :invitation_token => @token)
+  helpdesk_email = Rails.configuration.branding[:organisation][:helpdesk_email]

```

Comment:
same thing here. switch to the newer syntax please

Reply:
don't think is needed to explain again